Resources/views/Collector/db.html.twig

```diff
@@ -447,9 +447,17 @@
             var targetElement = document.getElementById(targetId);
 
             if (targetElement.style.display != 'block') {
-                Sfjs.load(targetId, link.href, null, function(xhr, el) {
-                    el.innerHTML = 'An error occurred while loading the query explanation.';
-                });
+                if (targetElement.getAttribute('data-sfurl') !== link.href) {
+                    fetch(link.href, {
+                        headers: {'X-Requested-With': 'XMLHttpRequest'}
+                    }).then(async function (response) {
+                        targetElement.innerHTML = await response.text()
+                        targetElement.setAttribute('data-sfurl', link.href)
+                        targetElement.classList.remove('loading')
```

Comment:
what is adding this class?

Reply:
I'm not sure about it. The Symfony code we were using was removing it, but it seems to be a left-over from older versions.